Dockerfile.base

```diff
@@ -8,7 +8,7 @@ ARG node_memory=8192
 WORKDIR /calypso
 ENV NPM_CONFIG_CACHE=/calypso/.cache
 ENV PERSISTENT_CACHE=true
-ENV READONLY_CACHE=false
+ENV GENERATE_CACHE_IMAGE=true
```

Comment:
Changing this for consistency. It's no longer good enough to just set `READONLY_CACHE`, and it wasn't easy to get the logic working in Kotlin/TeamCity.